autodepgraph/graph.py

```diff
@@ -85,6 +85,9 @@ def add_node(self, node_for_adding, **attr):
             'autodepgraph.node_functions.calibration_functions' +
             '.NotImplementedCalibration')
 
+        attr['calibrate_function_args'] = attr.get(
```

Comment:
I don't know how this function works but it might make more sense to have the default be an empty list of args. 

Also, I assume you don't want `calibrate_function_args` but instead `calibrate_function_kwargs`.

Reply:
yes, you're right.